src/components/playButton/PlayButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Icon } from '../icon'
+import { PlayButtonProps } from './PlayButton.types'
+import { ButtonWrapper } from './PlayButton.styles'
+
+export const PlayButton: React.FC<PlayButtonProps> = () => {
+  return (
+    <ButtonWrapper onClick={() => alert('clicked')}>
+      <Icon name="play-icon" variant="white" height="80px" width="80px" />
```

Comment:
Can you use the size from the theme's space scale instead of hard coding the size here as 80px. 
Also, on mobile it needs to scale down to 40px (xl_mob on the spacing scale). the Icon component uses styled-system layout props so you can define the size at each breakpoint (mobile first media queries).

for example: 

```
 <Icon 
    name="play-icon" 
    variant="white" 
    height={{default: "xl_mob", md: "xxl"}} 
    width={{default: "xl_mob", md: "xxl"}}  
/>
```